### PR TITLE
first part

### PR DESCRIPTION
**Pull request by belozerovany**

### CODE REVIEW

**Comment by belozerovany** [comment]
@uPhoenix  Please check the first part game without "vuex js "and "TDD"

File: src/components/MainMenu.vue [hunk 1]
```diff
@@ -10,13 +10,17 @@
    <div>
       <router-link to="/load" class="button button-huge block-mobile">Load Game</router-link>
    </div>
+   <div>
+      <router-link to="/teststore" class="button button-huge block-mobile">TEST STORE</router-link>
```
Comment (uPhoenix) [inline src/components/MainMenu.vue]: No test stuff in production

**Review by uPhoenix — CHANGES_REQUESTED** [review]

**Comment by belozerovany** [comment]
@uPhoenix  hello pleas check part without test part

**Comment by belozerovany** [comment]
@uPhoenix  
Good day.
Zahar pleas check  our project, we are added unit tests

**Pull request merged**